### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by brittanystoroz**
This adds an `ember fxos:validate` command that uses the [firefox-app-validator-manifest](https://github.com/mozilla/firefox-app-validator-manifest) module to validate the processed manifest.webapp file in the /dist directory. (Readme includes a note that the validator is still a WIP and may not catch all errors)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-npm install ember-cli-fxos --save-dev
-```
-
-## Creating Your Manifest File
+## Configuration
+### Manifest File
 Every FireFox OS application requires a [manifest file](https://developer.mozilla.org/en-US/Apps/Build/Manifest) in the root of your project.
 
-If you are using Ember CLI >= 0.1.5, the manifest file has already been created for you. If you are using an older version of Ember CLI, you can generate this file with the following command:
+After installing the addon, you'll see a `manifest.webapp` file was created in your application's /public directory. Any values that start with `@@` (e.g. `@@appName`) will be pulled from your `package.json` at build time. Reference the MDN link above for additional information on configuring your manifest file.
```
Comment (tofumatt) [inline README.md]: A very minor nitpick, but I found the wording here a bit hard to follow. What about:

"This addon creates a `manifest.webapp` in your application's `/public` directory."

Also can't hurt to just add the link again where it says MDN link instead of making the user look for it ;-)

File: README.md [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
 
-```
-ember g fxos-manifest
-```
+## Validation
+After running `ember build`, your final manifest file will be compiled into the /dist directory. To validate the manifest file and check for any errors or warnings, run:
```
Comment (tofumatt) [inline README.md]: Nitpick, but usually folders like dist are marked as code.

**Comment by tofumatt** [comment]
Looks good to me, not sure why Travis can't run the tests, but I'll try to re run them.

**Comment by tofumatt** [comment]
Oh, looks like this might fail if the public directory doesn't already exist. That's what travis seems to be complaining about. Maybe create the public directory if it doesn't already exist?

**Comment by brittanystoroz** [comment]
@tofumatt actually our builds have been failing on that since day one O_O Ha. It's looking for a /public dir at the root of the *addon* (which will never exist. and shouldn't. so we don't want to create it.) When that code is run within the context of an app, it maps to the correct /public dir (the one at the root of the app). Not sure why travis would be handling it differently. Will investigate.

**Comment by brittanystoroz** [comment]
Actually I guess I see some addons stubbed with empty folders of .gitkeep files. I'd like to go that route for now until I have some time to look through issues on the ember-cli project. Stubbing the public dir won't do any harm.

**Comment by tofumatt** [comment]
Ohhhhh whoops. But yeah, let's try to fix it, maybe even separately if this
works outside of travis. No need to stall this PR if the travis failure
isn't directly related to it.

-tofumatt
